cli/src/commands/init.rs

```diff
@@ -90,9 +117,79 @@ fn prompt_group() -> io::Result<Option<String>> {
         return Ok(None);
     }
 
-    let group: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("Project Group (default: none)")
-        .interact_text()?;
+    let group: String =
+        Input::new().with_prompt("Project Group (default: none)").interact_text()?;
 
     Ok(Some(group))
 }
+
+/// Interactively ask for missing lockfile information.
+fn prompt_lockfile(
+    cli_lockfile: Option<&String>,
+    cli_lockfile_type: Option<&String>,
+) -> io::Result<(String, Option<String>)> {
+    if let Some(lockfile) = cli_lockfile {
+        return Ok((lockfile.clone(), cli_lockfile_type.cloned()));
+    }
+
+    // Prompt for lockfile name.
+    let lockfile = prompt_lockfile_name()?;
+
+    // Try to determine lockfile name from known lockfiles.
+    for format in LockfileFormat::iter() {
+        if format.parser().is_path_lockfile(Path::new(&lockfile)) {
+            let lockfile_type = format.name().to_owned();
+            return Ok((lockfile, Some(lockfile_type)));
+        }
+    }
+
+    // Prompt for lockfile type.
+    let lockfile_type = prompt_lockfile_type()?;
+
+    Ok((lockfile, Some(lockfile_type)))
+}
+
+/// Ask for the lockfile name.
+fn prompt_lockfile_name() -> io::Result<String> {
+    // Find all known lockfiles in the currenty directory.
+    let mut lockfiles: Vec<_> = fs::read_dir("./")?
+        .into_iter()
+        .flatten()
+        .filter(|entry| {
+            LockfileFormat::iter().any(|format| format.parser().is_path_lockfile(&entry.path()))
+        })
+        .flat_map(|entry| entry.file_name().to_str().map(str::to_owned))
+        .collect();
+
+    // Prompt if a lockfile was found.
+    if !lockfiles.is_empty() {
+        // Add choice to specify an unknown lockfile.
+        lockfiles.push(String::from("other"));
+
+        // Ask user for lockfile.
+        let index = FuzzySelect::new()
+            .with_prompt("Select your project's lockfile")
+            .items(&lockfiles)
+            .interact()?;
+
+        // Return selected lockfile unless `other` was chosen.
+        if index + 1 != lockfiles.len() {
+            return Ok(lockfiles.remove(index));
+        }
+    }
+
+    // Prompt for lockfile name if none was selected.
+    Input::new().with_prompt("Project lockfile name").interact_text()
```

Comment:
This might be clearer if it asked for the `path` rather than the `name` so that the user doesn't think this is just some arbitrary name.

```suggestion
    Input::new().with_prompt("Project lockfile path").interact_text()
```